docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3.2'
+
+services:
+  factomd:
+    network_mode: host
+    container_name: factomd_container
+    image: factominc/factomd:latest
+    entrypoint: /go/bin/factomd
+    command: '--count=5 --net=alot+ --blktime=15 --faulttimeout=20000 --enablenet=false --network=LOCAL --startdelay=2'
+    ports:
+      - "8110:8110"
+      - "8109:8109"
+      - "8108:8108"
+      - "8090:8090"
+      - "8088:8088"
+    volumes:
+      - factomd_volume:/root/.factom/
+
+  walletd:
+    network_mode: host
+    container_name: walletd_container
+    image: factominc/factom-walletd:v2.2.14-alpine
+    ports:
+      - "8089:8089"
+    volumes:
+      - walletd_volume:/root/.factom/
+    depends_on:
+      - factomd
+
+  pegnet:
+    network_mode: host
+    container_name: pegnet_container
+    build: .
+    volumes:
+      - .:/go/src/github.com/pegnet/pegnet
+      - ./defaultconfig.ini:/root/.pegnet/defaultconfig.ini:ro
```

Comment:
I am not a Docker expert so I may be not understanding correctly, but in your Dockerfile you already copied that config file, so is it necessary to link this file via a volume?

Reply:
I do it so I can Edit the default config on the fly ( this allows you to not rebuild the image every time you wanna change that file)

Reply:
Oh ok, so does it override the file copied in the Dockerfile?

Reply:
yes, Dockerfile generates the base Image, then docker-compose does magic so we do not need to keep rebuilding the image every change.